js/src/package.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import Details from './lib/Details';
+
+const id = process.env.NODE_ENV == 'production'
+  ? location.pathname.split('/').slice(3).join('/')
```

Comment:
package names can contain a `/` (scoped packages). Is there a better way to get `"/en/package/XXX"` where XXX can contain `/`?